.github/workflows/build-iso.yml

```
@@ -21,79 +18,85 @@
+          compression-level: 0
+
+      - name: Get Artifact URL
+        id: get_iso_artifact
+        run: |
+          artifacts=$(curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/artifacts")
+          iso_url=$(echo "$artifacts" | jq -r '.artifacts[0].archive_download_url')
+          echo "iso_url=$iso_url" >> $GITHUB_ENV
+
+      - name: Generate Changelog
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2129:style:4:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)